module/support/logging/ConsoleLogHandler/src/ConsoleLogHandler.cpp

```diff
@@ -98,19 +105,20 @@ namespace module::support::logging {
             // If we know where this log message came from, we display that
             if (message.task != nullptr) {
                 // Get our reactor name
-                std::string reactor = message.task->identifier[1];
+                std::string reactor = message.task->identifiers.reactor;
 
                 // Strip to the last semicolon if we have one
                 size_t lastC = reactor.find_last_of(':');
                 reactor      = lastC == std::string::npos ? reactor : reactor.substr(lastC + 1);
 
                 // This is our source
                 source = reactor + " "
-                         + (message.task->identifier[0].empty() ? "" : "- " + message.task->identifier[0] + " ");
+                         + (message.task->identifiers.name.empty() ? "" : "- " + message.task->identifiers.name + " ");
             }
 
             // Output the level
             switch (message.level) {
+                case NUClear::UNKNOWN: std::cerr << source << ": "; break;
```

Comment:
I'd fold this in with fatal as if you get unknown something weird/wrong has happened